### PR TITLE
Updated import locations

### PR DESCRIPTION
**Pull request by oscargus**
#### Brief description of what is fixed or changed

Import documentation examples from as high level as possible.

#### Other comments
Must have started on an incorrect branch... Will fix merge conflicts online.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/functions/elementary/complexes.py [hunk 1]
```diff
@@ -696,16 +696,21 @@ def _eval_rewrite_as_conjugate(self, arg, **kwargs):
...
-    $(-\pi, \pi]$.  For a positive number, the argument is always $0$.
+    r"""
+    Returns the argument (in radians) of a complex number. The argument is
+    evaluated in consistent convention with ``atan2`` where the branch-cut is
+    taken along the negative real axis and ``arg(z)`` is in the interval
+    $(-\pi,\pi]$. For a positive number, the argument is always 0; the
+    argument of a negative number is $\pi$; and the argument of 0
+    is undefined and returns ``nan``. So the ``arg`` function will never nest
+    greater than 3 levels since at the 4th application, the result must be
+    nan; for a real number, nan is returned on the 3rd application.
```
Comment (oscarbenjamin) [inline sympy/functions/elementary/complexes.py:707]: Is it necessary to say this?

Comment (oscargus) [reply sympy/functions/elementary/complexes.py:707]: That is nothing that I have added on purpose (and wouldn't know), so probably a consequence of me starting from the wrong branch...

I'll see if I can get rid of it.

Comment (oscargus) [reply sympy/functions/elementary/complexes.py:707]: That doesn't seem to be in the latest (after I fixed the merge conflicts), only some formatting of numbers to latex.

**Comment by oscargus** [comment]
Make sure that the latest version is checked. There were quite a few "changed" files removed after resolving the conflicts...

File: sympy/combinatorics/perm_groups.py [hunk 2]
```diff
@@ -209,8 +208,7 @@ def equals(self, other):
         Examples
         ========
 
-        >>> from sympy.combinatorics import Permutation
-        >>> from sympy.combinatorics.perm_groups import PermutationGroup
+        >>> from sympy.combinatorics import , PermutationGroup
```
Comment (oscargus) [inline sympy/combinatorics/perm_groups.py:211]:
```suggestion
        >>> from sympy.combinatorics import Permuation, PermutationGroup
```

File: sympy/functions/elementary/piecewise.py [hunk 3]
```diff
@@ -653,8 +654,8 @@ def _intervals(self, sym, err_on_Eq=False):
         """Return a bool and a message (when bool is False), else a
```
Comment (oscargus) [inline sympy/functions/elementary/piecewise.py:654]:
```suggestion
        r"""Return a bool and a message (when bool is False), else a
```

File: sympy/combinatorics/permutations.py [hunk 4]
```diff
@@ -181,7 +181,7 @@ def _af_pow(a, n):
     Examples
     ========
 
-    >>> from sympy.combinatorics.permutations import Permutation, _af_pow
+    >>> from sympy.combinatorics import Permutation, _af_pow
```
Comment (oscargus) [inline sympy/combinatorics/permutations.py:184]:
```suggestion
    >>> from sympy.combinatorics import Permutation
    >>> from sympy.combinatorics.permutations import _af_pow
```

**Pull request merged**